src/core/router/net_db/impl.h

```diff
@@ -222,6 +222,12 @@ class NetDb {
       const IdentHash& destination,
       const std::set<IdentHash>& excluded) const;
 
+  bool IsRunning() const {
```

Comment:
I don't see the point of this function (it's shouldn't be needed for a unit-test either).